### PR TITLE
Add initial setup

### PR DESCRIPTION
**Pull request by jlberbesi**
Hello code reviewer,

I have added the actions below:

- Added linters 
- Addded gitflow
- Added react initial setup.

### CODE REVIEW

**Comment by addod19** [comment]
## Hi @jlberbesi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the README file is unprofessional and lacks all the relevant headers. For a more professional README file visit this [template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by jlberbesi** [comment]
@addod19 A new README file was added. Thanks for your feedback.

File: README.md [hunk 1]
```diff
@@ -1,70 +1,179 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 José Luis Berbesi
+
+- GitHub: [@jlberbesi](https://github.com/jlberbesi)
+- Twitter: [@imberbesi](https://twitter.com/jlberbesi)
```
Comment (Efiamotu-1) [inline README.md]: - [ ] This link is currently broken, I kindly suggest you provided the appropriate link to your `Twitter` handle. it is unprofessional to leave broken links in your readme file 👍

File: README.md [hunk 2]
```diff
@@ -1,70 +1,179 @@
...
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add popup window for reservations 😁**
+- **Add Episodes section 👌**
+- **Add more characters 😎**
+
+## Walkthrough <a name="walkthrough"></a>
+
+You can watch the video description of the project [here](https://---).
```
Comment (Efiamotu-1) [inline README.md]: - [ ] This link is currently broken, I kindly suggest you provided the appropriate link. it is unprofessional to leave broken links in your readme file 👍 
- [ ] THis section is also not a mandatory section so you can remove it. make sure to remove it from the headers at the top of your readme file as all

File: README.md [hunk 3]
```diff
@@ -1,70 +1,179 @@
...
+We would like to thank Microverse for giving us the opportunity to learn and grow as developers and also we like to thank our families, they are all our support. 🌟
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE) licensed.
```
Comment (Efiamotu-1) [inline README.md:173]: - [ ] This link is currently broken, I kindly suggest you provided the appropriate link to your License file. it is unprofessional to leave broken links in your readme file 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @jlberbesi , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Correct use of Gitflow
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (jlberbesi) [reply README.md]: Wait I just tested them and all of them are working . Weird

**Review by Efiamotu-1 — APPROVED** [review]
Hi @jlberbesi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

image

## Highlights :dart: 
* Correct Gitflow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**